src/composed/signed_key/public.rs

```diff
@@ -83,6 +83,7 @@ impl SignedPublicKey {
         mut public_subkeys: Vec<SignedPublicSubKey>,
     ) -> Self {
         public_subkeys.retain(|key| {
+            // FIXME: is it useful to make subkey retention depend on unvalidated signatures?
```

Comment:
issue?

Reply:
right, this is of course unrelated to v6. moved to https://github.com/rpgp/rpgp/issues/383